gravitee-apim-console-webui/src/components/gio-side-nav/gio-side-nav.harness.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentHarness } from '@angular/cdk/testing';
+
+export class GioSideNavHarness extends ComponentHarness {
+  public static hostSelector = '.ll';
```

Comment:
why not `gio-side-nav` ?

Reply:
... I'm just going to delete the `GioSideNavHarness`. It's not useful for my tests.